### ISSUE TITLE
ligand_expression_dotplot and ligand_differential_expression_heatmap are missing in Seurat_wrapper output

### ISSUE THREAD

**Issue by mlin2017** [saeyslab/nichenetr#121]
Hi @browaeysrobin,

Thank you for such a great tool!

I followed the step-by-step tutorial to analyze a Seurat object and it worked well. I switched to the Seurat Wrapper recently, and it is also great, except that ligand_expression_dotplot and ligand_differential_expression_heatmap are missing in the output from nichenet_seuratobj_aggregate(), shown below:
![Screen Shot 2022-01-25 at 6 04 15 PM](https://user-images.githubusercontent.com/25777670/151075610-4c24dd42-307b-4f2a-a427-a20c4d9b8f23.png)

Should I change any settings or there are some updates on this wrapper?

Thanks!

**Comment by browaeysrobin** [comment]
Hi @mlin2017 

In the old wrapper function, these elements are not included. In the newer version, they should be. Can you check this after updating nichenetr? And let me know?

**Comment by mlin2017** [comment]
Thank you @browaeysrobin!

Yes, I can see those outputs now after updating nichenetr. Thanks!

It gives me some warning messages  shown below. Should I make any changes? 

"Warning messages:
1: In nichenet_seuratobj_aggregate(seurat_obj = seuratObj, receiver = "Epithelial_1",  :
  Seurat object is result from the Seurat integration workflow. Make sure that the way of defining expressed and differentially expressed genes in this wrapper is appropriate for your integrated data.
2: In if (length(sender) > 1 | (length(sender) == 1 & sender != "undefined")) { :
  the condition has length > 1 and only the first element will be used"

The first warning shows that NicheNet correctly detected that I used an integrated Seurat object, but I specified the "assay_oi="RNA"". Is there anything else I need to pay attention to here? 

For the second warning, I included a list of sender cells here, and probably that is why it triggered this warning. I checked the outputs, and it seems that all sender cells were included in the analysis (as they all show up on the plots),not "only the first element will be used" as indicated in the warning message. Not sure this warning message is for a future version where only one sender cell or "all"/"undefined" would be the valid input for parameter "sender"...

Thanks!
